### ISSUE TITLE
Collapse git and openssl build/test jobs into one

### ISSUE THREAD

**Issue by rsbeckerca** [ituglib/ituglib-build#12]
The performance of OSSNS does not give advantage to running git builds in parallel. These should be folded into a single job.

**Comment by rsbeckerca** [comment]
This should be fixed before git 2.32 and OpenSSL 1.1.1l and/or OpenSSL 3.0.0-beta1

**Comment by rsbeckerca** [comment]
OpenSSL 3.0 now builds in one job.

**Comment by rsbeckerca** [comment]
Split off OpenSSL into its own case #15.
